Sources/BraveWallet/Crypto/Transaction Confirmations/PendingTransactionView.swift

```diff
@@ -0,0 +1,527 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BigNumber
+import Strings
+import DesignSystem
+
+struct PendingTransactionView: View {
+  @ObservedObject var confirmationStore: TransactionConfirmationStore
+  let networkStore: NetworkStore
+  let keyringStore: KeyringStore
+  @Binding var isShowingGas: Bool
+  @Binding var isShowingAdvancedSettings: Bool
+  
+  var onClose: () -> Void
```

Comment:
nit: we use `onDismiss` throughout wallet

Reply:
fixed in [29dc327](https://github.com/brave/brave-ios/pull/6960/commits/29dc3270f2191a8677bd908927acd01b0fc1f7de)